changelog.md

```diff
@@ -7,17 +7,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 # [v6.0.0]()
 
 ### Added
-* Added `sdk.media` class to handle and emit media related tasks
-* Added `sdk.media.on(...)` event emitter
-* Added permissions and device management housed in `sdk.media`
+* Added `sdk.media` class to handle and emit media related tasks. See [media documentation](doc/media.md) for more details
+  * `sdk.media.on(...)` event emitter. 
+  * permissions and device management housed in `sdk.media`
 
 ### Changed
-* Started migrating docs to use typescript definitions
+* Migrating docs to use typescript definitions
 
 ### Breaking Change
-* Updated configuration options for constructing an SDK instance
-* Moved `sdk.createMedia(opts)` to `sdk.media.startMedia(opts)`
+* Updated configuration options for constructing an SDK instance (see [docs](doc/index.md) for new config)
+  * Removed configuration option `iceServers: IceServerConfiguration[]` and `iceTransportPolicy`
+  * Moved defaults into a nested `config.default = {}` object
+* Changed `IAcceptSessionRequest`, `IEndSessionRequest`, and `ISessionMuteRequest` interfaces to require a `sessionId` 
+  in place of the non-descriptive `id` field. This contract change will impact calls to:
+    * `sdk.acceptSession({sessionId: string;})`
+    * `sdk.endSession({sessionId: string;})`
+    * `sdk.setVideoMute({sessionId: string;})`
+    * `sdk.setAudioMute({sessionId: string;})`
+    * See [docs](doc/index.md) for method parameters
+
+* Moved & renamed `sdk.createMedia(opts)` to `sdk.media.startMedia(opts)`
 * Moved `sdk.getDisplayMedia(opts)` to `sdk.media.startDisplayMedia(opts)`
+* `sdk.updateOutputDevice()` will now log a warning and do nothing if called in an unsupported browser.
+  Old behavior was to throw an error.
+* Updated `sdk.on('handledIncomingRtcSession', evt => {})` typing to be a `sessionId: string` and not an `IExtendedMediaSession`. 
+  Event already emitted the `sessionId`. This fixes the typing. 
```

Comment:
These are some notable breaking changes I made while working through this ticket.